Apps/phwindowsdefenderatp/readme.html

```diff
@@ -204,15 +204,15 @@ <h2>Explanation of Test Connectivity Workflow for Interactive auth and Non Inter
+                <li>The authorization code generated in the above step is used by the connectivity to make the next API call to generate the [access_token] and refresh_token pair. The generated authorization code, [access_token], and [refresh_token] are stored in the state file of the app on the Phantom server.</li>
+                <li>The authorization code can be used only once to generate the pair of [access_token] and [refresh_token]. If the [access_token] expires then, the [refresh_token] is used internally automatically by the application to re-generate the [access_token] by making the corresponding API call. This entire autonomous workflow will seamlessly work until the [refresh_token] does not get expired. Once the [refresh_token] expires, the user will have to run the Test Connectivity action once again to generate the authorization code followed by the generation of an entirely fresh pair of [access_token] and [refresh_token]. The default expiration time for the [access_token] is 1 hour and that of the [refresh_token] is 90 days. For more details visit <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-configurable-token-lifetimes" target="_blank">AD Configurable Token Lifetimes</a></li>
                 <li>The successful run of the Test Connectivity ensures that a valid pair of [access_token] and [refresh_token] has been generated and stored in the app's state file. These tokens will be used in all the actions' execution flow to authorize their API calls to the Defender ATP instance.</li>
               </ul>
               <li>Run the Test Connectivity action for Non Interactive method.</li>
               <ul>
                 <li>Internally, the application authenticates to Azure AD token issuance endpoint and requests an [access_token] then it will generate the [access_token].</li>
                 <li>The [access_token] generated in the above step is used by the test connectivity to make the next API call to verify the [access_token]. The generated [access_token] is stored in the state file of the app on the Phantom server.</li>
-                <li>If the [access_token] gets expired then, application will automatically re-generate the [access_token] by making the corresponding API call.</li>
+                <li>If the [access_token] expires then, application will automatically re-generate the [access_token] by making the corresponding API call.</li>
```

Comment:
```suggestion
                <li>If the [access_token] expires, then application will automatically re-generate the [access_token] by making the corresponding API call.</li>
```

Reply:
Hi @alexa-phantom, I have made the changes suggested by you.